deposit/src/main/java/edu/unc/lib/deposit/normalize/DirectoryToBagJob.java

```diff
@@ -43,6 +55,8 @@
  * @author lfarrell
  */
 public class DirectoryToBagJob extends AbstractFileServerToBagJob {
+	private static final Logger log = LoggerFactory.getLogger(IngestDeposit.class);
```

Comment:
classname should match the name of the class you are logging for (DirectoryToBagJob.class), otherwise the logging statements will be listed as coming from the wrong class.